README.md

```diff
@@ -29,7 +29,14 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+function Instructor(name){
+   this.name = name;
+   this.givesHomework = function(assignment){
```

Comment:
Almost! But you would need to use `this.name` instead of name in the method `givesHomework`:   

```
this.givesHomework = function(assignment){
    console.log(this.name + " gives the students " + assignment + " for Friday's homework.");
  }
```